ledger/src/blockstore_db.rs

```diff
@@ -1995,13 +1996,18 @@ pub struct LedgerColumnOptions {
     // Determine the way to compress column families which are eligible for
     // compression.
     pub compression_type: BlockstoreCompressionType,
+
+    // Control the number of performance samples in a million for RocksDB reads
+    // and writes.
+    pub perf_context_samples_in_1m: u32,
```

Comment:
That's a good point.  Originally it was n / 1000 before this PR, but I want to give it finer precision.

How about we simplify this to 1 / N (instead of n / 1,000,000) and we have a variable to control N, and maybe call it `perf_sample_interval`?

Reply:
> How about we simplify this to 1 / N (instead of n / 1,000,000) and we have a variable to control N, and maybe call it perf_sample_interval?

that was what I am suggesting with rate/cadence/freq.
if you want to name it "interval" that is fine too.

Reply:
Sounds good.  The reason behind this is that rate/freq gives us an impression that a higher value means more samples while it is actually the reverse. I googled a bit and it seems interval is a good term for describing the inverse of frequency.

Reply:
Changed to 1 / N concept and name it `rocks_perf_sample_interval`.